test/ui-test/testSuites/suite_communities/tst_communityNavigation/test.py

```diff
@@ -0,0 +1,8 @@
+source(findFile('scripts', 'python/bdd.py'))
+
+setupHooks('bdd_hooks.py')
+collectStepDefinitions('./steps', '../shared/steps/', '../../global_shared/steps/', '../../suite_messaging/shared/steps/')
```

Comment:
Is this reference to `suite_messaging` really needed here?
```suggestion
collectStepDefinitions('./steps', '../shared/steps/', '../../global_shared/steps/')
```